shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -197,9 +205,8 @@ - (instancetype)initWithBridge:(fml::WeakPtr<flutter::AccessibilityBridge>)bridg
 
 - (void)dealloc {
   for (SemanticsObject* child in _children) {
-    child.parent = nil;
+    [child privateSetParent:nil];
   }
-  [_children removeAllObjects];
```

Comment:
Another possibility is that someone is holding onto the `SemanticsObject.children`, if they were doing that that I don't think we should be clearing it out.  I'll put it back in since it doesn't have direct impact on the issue.